### PR TITLE
Crud

### PR DESCRIPTION
**Pull request by yukiej**
@jonpaul108 Let me know if you have any feedback on anything, thank you! 

p.s. this should be the final and correct pull request :) sorry for the spam

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -13,13 +13,53 @@ app.use(function(req, res, next) {
...
+      return handleError(err);
+    } else {
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(data));
+    }
   })
 })
+//create new stock
+app.post('/api/volumes/symbols/:id', (req, res) => {
+  let newStock = new PriceVolume({ ...req.params });
```
Comment (jonpaul108) [inline server/index.js]: This was cool

File: server/index.js [hunk 2]
```diff
@@ -13,13 +13,53 @@ app.use(function(req, res, next) {
...
+  newStock.save((err, doc) => {
+    if (err) {
+      return handleError(err);
+    } else {
+      console.log("New stock created: ", doc);
```
Comment (jonpaul108) [inline server/index.js]: is this console.log a test or do you want to keep it for the developer?
consider deleting if it was left in.

File: database/index.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 const mongoose = require('mongoose');
 const mongoUri = 'mongodb://localhost/price_volume_chart_service';
 
-const db = mongoose.connect(mongoUri);
+const db = mongoose.connect(mongoUri, (err) => {
+  if (err) {
+    throw err;
+  } else {
```
Comment (jonpaul108) [inline database/index.js]: good call on that update

File: server/index.js (hunk 1 above)
Comment (yukiej) [reply server/index.js]: Thank you! :)

File: server/index.js (hunk 2 above)
Comment (yukiej) [reply server/index.js]: Good call! Edited to get rid of all the console logs.

**Pull request merged**